### PR TITLE
[DO_NOT_MERGE] Fix PipeLinePMDAO

### PR DESCRIPTION
**Pull request by hchoura**

### CODE REVIEW

File: src/foam/nanos/pm/NullPM.js [hunk 1]
```diff
@@ -29,7 +29,16 @@ foam.CLASS({
           type: 'X'
         }
       ],
-      javaCode: '/*nop*/'
+      javaCode: `
```
Comment (jlhughes) [inline src/foam/nanos/pm/NullPM.js:32]: @hchoura what's reason for an implementation?

Comment (hchoura) [reply src/foam/nanos/pm/NullPM.js:32]: To activate the log.

Comment (jlhughes) [reply src/foam/nanos/pm/NullPM.js:32]: Please explain.  If you need PMs for every create, then change the percentage in the nspec, it's presently set to 1%, change it 100. 

foam/nanos/pm/services.jrl

p({
  class:"foam.nanos.boot.NSpec",
  name:"PM",description:"Configurable PM Factory",
  authenticate:false,
  pm:false,
  service:{
    class:"foam.nanos.pm.PMFactory",
    percentage:1
  }
})

File: src/foam/nanos/pm/PMInfo.js [hunk 2]
```diff
@@ -80,6 +80,7 @@ foam.CLASS({
 
   methods: [
     {
+      synchronized: true,
```
Comment (jlhughes) [inline src/foam/nanos/pm/PMInfo.js:83]: What issue are you seeing? synchronization in PMs is the last thing we need as they need to be super fast.

**Pull request closed without merging**